jax/BUILD

```diff
@@ -84,6 +84,30 @@ config_setting(
     },
 )
 
+# Which major version of CUDA to target
+string_flag(
```

Comment:
I wonder if the source of truth of this flag should go with the other CUDA detection stuff in TSL.